comps/cores/mega/gateway.py

```diff
@@ -815,8 +819,13 @@ def _handle_message(self, messages):
                         image_list = [
                             item["image_url"]["url"] for item in message["content"] if item["type"] == "image_url"
                         ]
+                        audios = [
+                            item["audio"] for item in message["content"] if item["type"] == "audio"
+                        ]
                         if image_list:
                             messages_dict[msg_role] = (text, image_list)
+                        elif audios:
```

Comment:
We probably want to allow a message to contain both an image AND an audio, for example if it's a follow-up query about an image but the query is in audio format.